app/tests/workstations_tests/test_templatetags.py

```diff
@@ -18,6 +18,7 @@ def test_workstation_query(settings):
     algorithm_job = AlgorithmJobFactory()
     config = WorkstationConfigFactory()
     archive_item = ArchiveItemFactory()
+    user = UserFactory()
```

Comment:
Whilst you're here could you try refactoring this to not use the DB? You should be able to switch to using the `build` methods of the factories.